app/views/placements/schools/placements/show.html.erb

```diff
@@ -42,6 +42,11 @@
           <% summary_list.with_row do |row| %>
             <% row.with_key(text: t(".attributes.placements.year_group")) %>
             <% row.with_value(text: t("placements.schools.placements.year_groups.#{@placement.year_group}")) %>
+            <% row.with_action(
```

Comment:
Should `year_group` also appear on the support version of placements show?
Might be worth breaking this out into a partial?

Reply:
@JamieCleare2525 yes ⭐ there's a card for this:

https://trello.com/c/3v9E3GOW/449-update-support-placement-view-to-include-phase-and-provider

Reply:
I went down this path and realised these two views are pretty out of sync, worth breaking down into its own task to normalise them with some design/product input.

https://trello.com/c/Z4mem3gv/452-create-details-partial-for-placement-show-pages